Src/App/Router.php

```diff
@@ -8,30 +8,25 @@ class Router
     /**
      * lead to the controller
      * @param $parameters
-     * @return mixed
+     * @return array
      * @throws \Exception
      */
     public function route($parameters)
     {
-
+        $controllerClassName = ucfirst($parameters['controller']) . 'Controller';
```

Comment:
`$parameters['controller']` may not exist, you should test it before

Reply:
done